Integrations/Digital_Defense_FrontlineVM/Digital_Defense_FrontlineVM.py

```diff
@@ -0,0 +1,795 @@
+    # Vulnerability statistics output:
+    vuln_stat_ec = {
+        'FrontlineVM(val.Object && val.Object == obj.Object)': vuln_stat_output
+    }
+    return {
+        'Type': entryTypes['note'],
+        'Contents': vuln_stat_output,
+        'ContentsFormat': formats['json'],
+        'HumanReadable': tableToMarkdown('FrontlineVM: Vulnerability Statisctics', vuln_stat_output, headers=vuln_stat_headers),
+        'EntryContext': vuln_stat_ec
+    }
+
+
+def get_vuln_outputs(vuln_list):
+    ''' Get and prepare output from list of raw vulnerability data '''
+    output = []
+    vuln_data_output = {}   # type: Dict
+    vuln_stat_output = {}   # type: Dict
+
+    vuln_severity_count = {
+        'critical': 0,
+        'high': 0,
+        'medium': 0,
+        'low': 0,
+        'trivial': 0,
+        'info': 0
+    }
+
+    vuln_stat_output = {}
+    vuln_stat_output['vulnerability-count'] = len(vuln_list)
+    vuln_data_list = []
+    ip_list = []
+
+    if len(vuln_list) > 0:
+        demisto.debug('FrontlineVM get_vuln_outputs -- iterating through vulns')
+    else:
+        demisto.debug('FrontlineVM get_vuln_outputs -- get vulns request returned empty')
+
+    # Condensing Vuln data for HumanReadable and EntryContext:
+    for vuln in vuln_list:
+        vuln_obj = create_vuln_obj(vuln)    # vulnerability data output
+        vuln_severity_count[str(vuln['severities']['ddi'])] += 1
+        vuln_data_list.append(vuln_obj)
+        ip_list.append(vuln.get('ip_address'))
+
+    # Include severity in vulnerability statistic header if severity exists:
+    vuln_stat_headers = ['vulnerability-count']
+    for severity_key, count_value in vuln_severity_count.items():
+        # Include severity level (critical, high, medium, etc...) in output header if vulnerability exists for that severity.
+        if count_value > 0:
+            vuln_stat_headers.append(str(severity_key) + "-severity-count")
+            vuln_stat_output[str(severity_key) + "-severity-count"] = count_value
+
+    vuln_data_output['Vulns'] = vuln_data_list
+    vuln_data_output['IPList'] = ip_list
+
+    # Format and generate output:
+    data_output = format_vuln_data_output(vuln_data_output, vuln_list, vuln_data_list)
+    output.append(data_output)
+
+    stat_output = format_vuln_stat_output(vuln_stat_output, vuln_stat_headers)
+    output.append(stat_output)
+    return output
+
+
+def get_host_id_from_ip_address(ip_address):
+    '''
+        Get host ID within Frontline.Cloud given IP address.
+        Host ID used to pull vulnerability data for that specific host.
+    '''
+    host = get_fvm_data(HOST_ENDPOINT, params={'_0_eq_host_ip_address': str(ip_address)})
+    if len(host) < 1:
+        msg = 'Host not found within Frontline.Cloud given host IP Address.'
+        demisto.debug('Frontline.Cloud get_host_id_from_ip_address -- ' + msg)
+        return_error("Error: " + msg)
+    return host[0].get('id')
```

Comment:
That particular endpoint (HOST_ENDPOINT) will always return a list of hosts, even if just one hosts is returned.
It just so happens for this case, we are querying for only one particular host with one particular ip address. So this will still return a list of host results, and in this case it is only one host.

Fixed with suggestions to change the variable names.